### PR TITLE
feat/create-an-endpoint-to-endgame

### PR DESCRIPTION
**Pull request by Alao-Abiodun**

### CODE REVIEW

File: server/src/controllers/game.controller.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ class GameController {
         .status(201)
         .send(response("Game created successfully", gameDBData.data, true));
     } catch (error) {
+      console.log(error);
```
[the remark this answers is not in the document]
Comment (Alao-Abiodun) [reply server/src/controllers/game.controller.js]: @eni4sure I have removed it, thanks, I was trying to test them if it's working the ZC_Core URL.

File: server/src/controllers/game.controller.js [hunk 2]
```diff
@@ -107,8 +107,46 @@ class GameController {
...
+
+      if (!user_id) {
+        return res
+          .status(400)
+          .send(response("User does not exist", null, false));
+      }
+
+      let permission;
+
+      const updated = await GameRepo.delete(game_id, {
```
Comment (eni4sure) [inline server/src/controllers/game.controller.js:129]: boss, this should be ``GameRepo.update`` not ``delete``

**Review by eni4sure — CHANGES_REQUESTED** [review]
Kindly fix this too

File: server/src/controllers/game.controller.js (hunk 2 above)
Comment (NdubuisiJr) [reply server/src/controllers/game.controller.js:129]: Yeah... you should be updating the game status.

**Pull request closed without merging**